src/locals/de.ts

```diff
@@ -53,4 +54,14 @@ export default {
     login_failed: 'Username oder Passwort ist falsch.',
     login_incomplete: 'Bitte geben Sie Username und Passwort an.',
   },
+  connection: {
+    server_unavailable: {
```

Comment:
Is it common practice to use the underscore here?

Reply:
is